tools/migrate_photo_to_odp.rb

```diff
@@ -0,0 +1,20 @@
+# Migrate all photos with open licenses (not 0,8,10) from the Static bucket to the Open bucket
+
+max_photo_id = LocalPhoto.last.id
+batch_size = 100000
+index = 0
+
+while index < max_photo_id
+	LocalPhoto.joins("LEFT JOIN flags ON (photos.id = flags.flaggable_id)").
+  where("flags.id IS NULL").
+  where("photos.file_prefix_id=2")
```

Comment:
This line needs a dot at the end, otherwise it will look like the statement ends here and a new statement begins on the next line starting with `where(` which is invalid syntax.

Same minor code formatting comments for this file - spaces before/after parens and operators, e.g. `where( "photos.file_prefix_id = 2" ).` on this line. The hard-coded prefix ID could be replaced with `FilePrefix.where(nprefix: "https://static.inaturalist.org/photos" ).first.id`